benchmarks/helics/BenchmarkFederate.hpp

```diff
@@ -118,15 +118,20 @@ class BenchmarkFederate {
         app;  //<! the CLI11 app object to use in derived classes
 
     // functions to be overridden by derived benchmark classes
-    /** set/override default base parameter values before arguments are parsed, and modify CLI11 options for default arguments*/
+    /** set/override default base parameter values before arguments are parsed, and modify CLI11
+     * options for default arguments*/
```

Comment:
If it gets split across multiple lines, is there an option to make it move `*/` to it's own line at the end?

Reply:
I haven't seen that sort of thing yet but will look into it more.